### PR TITLE
Github workflow cleanup

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: .github/workflows/alpha.yaml [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+      uses: actions/checkout@v2
+
+    - uses: azure/docker-login@v1
+      env:
+        DOCKER_CONFIG: ./migrations/.docker
+      with:
+        username: ${{ secrets.DOCKERHUB_USER }}
+        password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+    - name: Get tags
```
Comment (divolgin) [inline .github/workflows/alpha.yaml:95]: this is not used

File: .github/workflows/alpha.yaml [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+
+      # TODO: migrate retry logic for pacts
+      # retry:
+      #   automatic:
+      #     # this command exiting with status 2 typically means that the yarn package failed to download
+      #     - exit_status: 2
+      #       limit: 5
+
+
+  build_web:
```
Comment (divolgin) [inline .github/workflows/alpha.yaml:140]: This  needs to run in tagged workflow as well

Comment (emosbaugh) [reply .github/workflows/alpha.yaml:140]: it does

File: .github/workflows/alpha.yaml [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+      with:
+        path: ${{ steps.go-cache-paths.outputs.go-build }}
+        key: ${{ runner.os }}-go-build-${{ hashFiles('**/go.sum') }}
+    - uses: actions/cache@v2
+      with:
+        path: ${{ steps.go-cache-paths.outputs.go-mod }}
+        key: ${{ runner.os }}-go-mod-${{ hashFiles('**/go.sum') }}
+
+    - name: Get tags
+      id: get_tag
```
Comment (divolgin) [inline .github/workflows/alpha.yaml:337]: this isn't used.  (i know you didn't add it)

**Review by divolgin — APPROVED** [review]

File: .github/workflows/alpha.yaml (hunk 3 above)
Comment (emosbaugh) [reply .github/workflows/alpha.yaml:337]: is this gonna be an empty string in this entire workflow? should i remove all references?

File: .github/workflows/alpha.yaml (hunk 2 above)
Comment (divolgin) [reply .github/workflows/alpha.yaml:136]: I didn't see `build-kotsadm` anywhere else

**Review by murphybytes — APPROVED** [review]
LGTM I'm assuming next steps would be to source the .image.env file that I'm generating and update the image reference tags here.

**Review by divolgin — APPROVED** [review]

File: .github/workflows/alpha.yaml (hunk 3 above)
Comment (divolgin) [reply .github/workflows/alpha.yaml]: yep

**Pull request merged**